Automattic/jetpack
# supercache: update homepage etc on post un-publish

Fixes #39698

## Proposed changes:

Call the usual `wp_cache_post_edit()` function for post unpublication events as well to ensure that the homepage and so on are correctly updated.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Enable and configure supercache
2. Publish a post
3. Confirm that homepage cache file exists and contains the post
4. Unpublish the post (e.g. by returning it to draft status)
5. Confirm that the homepage cache file has been recreated and no longer contains the post
